artifacts/debian/debian.go

```diff
@@ -0,0 +1,117 @@
+package debian
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/containerdisks/pkg/api"
+	"kubevirt.io/containerdisks/pkg/architecture"
+	"kubevirt.io/containerdisks/pkg/docs"
+	"kubevirt.io/containerdisks/pkg/hashsum"
+	"kubevirt.io/containerdisks/pkg/http"
+	"kubevirt.io/containerdisks/pkg/tests"
+)
+
+type debian struct {
+	Arch            string
+	Version         string
+	VersionName     string
+	getter          http.Getter
+	ExampleUserData *docs.UserData
+	envVariables    map[string]string
+}
+
+const (
+	baseURLFmt   = "https://cloud.debian.org/images/cloud/%s/latest/"
```

Comment:
We should fetch the versioned images instead of `latest` to have this information available in tags. Otherwise the image will never get rebuilt once it was built.

Reply:
Added additional tag.